examples/README.md

```diff
@@ -18,13 +18,13 @@
            <img align="absmiddle" height="20" src="https://gitpod.io/button/open-in-gitpod.svg"/>
          </a> 
   - Learn how to write a gRPC service with Armeria gRPC module.
-  - See [Running a gRPC service](https://line.github.io/armeria/docs/server-grpc) and
-    [Calling a gRPC service](https://line.github.io/armeria/docs/client-grpc).
+  - See [Running a gRPC service](https://armeria.dev/docs/server-grpc) and
+    [Calling a gRPC service](https://armeria.dev/docs/client-grpc).
     
 - `grpc-kotlin`
   - Learn how to write a gRPC service with Armeria gRPC module (Kotlin).
-  - See [Running a gRPC service](https://line.github.io/armeria/docs/server-grpc) and
-    [Calling a gRPC service](https://line.github.io/armeria/docs/client-grpc).
+  - See [Running a gRPC service](https://armeria.dev/docs/server-grpc) and
+    [Calling a gRPC service](https://armeria.dev/docs/client-grpc).
 
 - `grpc-reactor` <a href="https://gitpod.io/#project=grpc-reactor/https://github.com/ikhoon/armeria-examples/tree/master/grpc-reactor/src/main/java/example/armeria/grpc/reactor/Main.java">
```

Comment:
Oops... could you mind changing s/`ikhoon/armeria-examples`/`line/armeria-examples`? 🙈

Reply:
😱